pkg/gvkcompare/gvkcompare.go

```diff
@@ -0,0 +1,276 @@
+package gvkcompare
```

Comment:
Perhaps the package named `gvk` would better support a broader range of tools in the future.
Within the package, suggest a `comparator.go` file would be intuitive.

Reply:
Actually, another question.  If only used by the plan controller, why is this a top level package instead of a sub-package under the plan controller package?